packages/license-kit/package.json

```diff
@@ -59,7 +59,7 @@
     "typescript": "^5.8.3"
   },
   "dependencies": {
-    "@callstack/react-native-legal-shared": "^0.2.0",
+    "@callstack/licenses": "^0.2.0",
```

Comment:
let's revert it here

let's first publish just new shared package, then we can do following PR that updates dependency here - the reason is that when publishing all the packages at once, there might be an error in the pipeline, because shared package is not published yet on npm

Reply:
Good idea, let's do it step-by-step this way. Addressed in f8c6036.